### ISSUE TITLE
King display incorrect

### ISSUE THREAD

**Issue by acheeseye** [acheeseye/ai-playing-checkers#4]
The following moves produces a red king that does an opposite direction jump from a king-transformer spot and an opposite direction move from a non-king-transformer spot:
92 79 57 68 79 57 55 68 90 79 68 90 103 77 44 55 86 75 55 68 75 62 33 44 77 66 53 79 88 75 44 55 105 92 79 105 75 64 51 77 101 88 68 79 88 66 55 77 116 103 105 116 99 86 116 90 86 75 79 92 110 99 90 79
After the red king leaves the king-transformer spot, the piece is displayed as a red man.
This may or may not be due to the piece leaving the king-transformation spot, but rather `board_update` or related functions are overwriting the board status thus altering the printed value.

TL;DR new red king functions as red king but drawn as red man.

**Comment by acheeseye** [comment]
Fixed: problem with not using `else if` statement and causing a overwrite after `board_update`.
92 79 57 68 79 57 55 68 90 79 68 90 103 77 44 55 86 75 55 68 75 62 33 44 77 66 53 79 88 75 44 55 105 92 79 105 75 64 51 77 101 88 68 79 88 66 55 77 116 103 105 116 99 86 116 90 86 75 79 92 110 99 90 79 99 86 31 44 57 31 79 68 31 53 40 66
This now produces both a red and black king that may move in all directions. The black king gets killed by a red man and is no longer displayed.
